broker/lib/models/catalog.js

```diff
@@ -36,4 +36,4 @@ class Catalog {
   }
 }
 
-module.exports = new Catalog();
\ No newline at end of file
+module.exports = new Catalog();
```

Comment:
Extra new line added. Suggesting the change as we need to change in other places as well. This is not the only change.